### PR TITLE
First Pass at adding Shorthand

### PR DESCRIPTION
**Pull request by bnb**
First pass at adding shorthand. There's still some more additions I'd like to make, but I figure those can come in a different PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,13 +39,42 @@ The meeting schedule, issue, etc can be configured with inouts to this action.
...
-- `issueTemplate`: The name of the issue template found in `.github/ISSUE_TEMPLATE`. Default: `meeting.md`
+- `issueTemplate`: The name of the issue template found in `.github/ISSUE_TEMPLATE`. Default: `meeting.md` 
 
 ### Default Issue Template
 
-The default issue template can be seen here: https://github.com/wesleytodd/meeting-maker/issues/34
+The default issue template can be seen in the ISSUE_TEMPLATES directory, as [meeting.md](https://github.com/pkgjs/meet/blob/main/.github/ISSUE_TEMPLATE/meeting.md). Additionally, you can find example issues of the latest version via the closed issues that are the result of CI tests.
 
-It is based off the one commonly used on across the Node.js Org, but any additions or improvements are welcome.
+Our default template is based off of the practices of the Node.js project's practices around meetings, but changes and enhancements are welcome.
```
Comment (rodion-arr) [inline README.md:48]: Is it correct to have "practices" twice in this sentence?

File: README.md [hunk 2]
```diff
@@ -39,13 +39,42 @@ The meeting schedule, issue, etc can be configured with inouts to this action.
...
+#### Shorthand in Your Custom Meeting Templates
+
+We provide some shorthand that you can use in your meeting templates. We'll automatically replace the shorthand with the relevant dynamic information.
+
+- Title:
+  - Shorthand: `<!-- title -->`
+  - Result: The issue's title.
+- Agenda Label:
+  - Shorthand: `<!-- agenda label -->`
+  - Result: The label for agenda items to be pulled from that youv'e defined in the Action's YAML configuration.
```
Comment (rodion-arr) [inline README.md:65]:
```suggestion
  - Result: The label for agenda items to be pulled from that you've defined in the Action's YAML configuration.
```

File: README.md [hunk 3]
```diff
@@ -39,13 +39,42 @@ The meeting schedule, issue, etc can be configured with inouts to this action.
...
+- Invitees:i
```
Comment (rodion-arr) [inline README.md:66]:
```suggestion
- Invitees:
```

File: README.md (hunk 1 above)
Comment (bnb) [reply README.md:48]:
```suggestion
Our default template is based off of the Node.js project's practices around meetings, but changes and enhancements are welcome.

```

**Review by joesepi — APPROVED** [review]
LGTM.

Perhaps we land this and if you can describe what you thought would be next steps in an issue, @bnb, I might have someone who can help implement those next steps. 🙌

**Pull request merged**